crates/wysiwyg/src/dom/action_list.rs

```diff
@@ -195,31 +195,31 @@ mod test {
     use super::*;
     use widestring::Utf16String;
 
-    type DAL = DomActionList<Utf16String>;
+    type Dal = DomActionList<Utf16String>;
```

Comment:
Clippy doesn't like capitalized acronyms for some reason, even if you do it snake case like D_A_L

Reply:
I agree `Dal` is a bit weird. We could also change the alias to something meaningful `Actions` or `ActionList`? Or get rid of the alias and create a local function that calls `DomActionList<Utf16String>::new`?

Reply:
Just noting for oversight over the Christmas-New year period:
This was updated and merged